docs/standard/analyzers/portability-analyzer.md

```diff
@@ -7,35 +7,58 @@ ms.assetid: 0375250f-5704-4993-a6d5-e21c499cea1e
+To use ApiPort console app, you need to download it from [ApiPort repository](https://github.com/microsoft/dotnet-apiport). It comes with both .NET Core version and .NET Framework version. You can  use "listTargets" command option to display the available target list, then pick target platforms by specifying -t or --target command option.
+
+### Analyze portability
+In Visual Studio to analyze your entire project, right-click on your project in **Solution Explorer** and select **Analyze Assembly Portability**. Otherwise, go to the **Analyze** menu and select **Analyze Assembly Portability**. From there, select your project’s executable or DLL.
 
 ![Portability Analyzer from Solution Explorer](./media/portability-analyzer/portability-solution-explorer.png)
 
-After running the analysis, you'll see your .NET Portability Report. Only types that are unsupported by a target platform appear in the list and you can review recommendations in the **Messages** tab in the **Error List**. You can also jump to problem areas directly from the **Messages** tab.
+If you want to use [ApiPort console app](https://github.com/Microsoft/dotnet-apiport/releases).
+
+* Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
+* To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
+* Run `\...\ApiPort.exe -?` to get more help
+
+### View and interpret portability result
+Only APIs that are unsupported by a Target Platform appear in the report. 
+After running the analysis in Visual Studio, you'll see your .NET Portability report pops up. 
 
 ![Portability Report](./media/portability-analyzer/portability-report.png)
 
-If you don’t want to use Visual Studio, you can use the Portability Analyzer from the command prompt. Just download the API Portability Analyzer from the [Microsoft/dotnet-apiport](https://github.com/Microsoft/dotnet-apiport/releases) repository.
+If you used [ApiPort console app](https://github.com/Microsoft/dotnet-apiport/releases), your .NET Portability report is saved as a file in your specified format, default is in an Excel file (*.xlsx*) in your current directory. 
 
-* Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
-* To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
+#### Portability Summary 
+This session of the report gives the assembly level overall view of the portability percentage.  As in above report example, it means that 89.74% of .NET Framework APIs the ConsoleAppFramework app used are available in .NET Core + Platform Extensions v2.2. If you run .NET Portability Analyzer tool against multiple assemblies, each of the assembly should have a row in the Portability Summary report.
+
+#### Details
+List the APIs missing from one of the Target Platforms. 
```

Comment:
you need to improve the formatting here. You can see how it's rendering right now:
https://review.docs.microsoft.com/en-us/dotnet/standard/analyzers/portability-analyzer?branch=pr-en-us-12986#details

Reply:
fixed